### PR TITLE
fix: workflow failed

### PR DESCRIPTION
**Pull request by sadanandl**
### Description

Lock file updated
PHPCS file added

### Checklist:
- [ ] My code is tested
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
-          coverage: none
-          tools: composer
+    - name: NPM Dependencies
+      if: always()
+      run: yarn install
 
-      - name: Install dependencies
-        run: composer config github-oauth.github.com ${{ secrets.PRIVATE_ACCESS_TOKEN }} && composer install --prefer-dist --no-suggest --no-progress
+    - name: Run phpcs
+      run: ./vendor/bin/phpcs --standard=phpcs.xml.dist
```
Comment (vrundakansara) [inline .github/workflows/ci.yml:48]: Replace .dist file with XML

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vrundakansara) [reply .github/workflows/ci.yml:48]: In that case you can remove it from ignore

**Review by sagarb90 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sadanandl) [reply .github/workflows/ci.yml:48]: We always used .dist file in free plugin.  You can see on master branch: https://github.com/brainstormforce/astra-sites/blob/master/phpcs.xml.dist

Why you deleted that file here: https://github.com/brainstormforce/astra-sites/pull/517/files#diff-05ae9cddcaec1e845771a7db224961439f83ef5939ec67d3a48744cb34d7e58bL1
Any reason?

**Review by vrundakansara — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vrundakansara) [reply .github/workflows/ci.yml]: @sadanandl - There is no such rule to do so, we did the same thing in the Pro plugin and other BSF plugins. We want to follow the same format.

**Pull request merged**